### ISSUE TITLE
Redesign UI using HTML + Spring

### ISSUE THREAD

**Issue by dgarijo** [dgarijo/Widoco#481]
After some thought, it seems like the UI in Swing is not very maintainable: there are differences in the look and feel in different systems and it looks clunky and old. Redesigning it in Swing is going to be a lot of work. Hence, I think a better idea is to design it in HTML and show it in a browser with Spring. In addition, this will help previsualizing the documentation, as we can serve the results internally. The client will also work with Docker this way.

**Comment by seralf** [comment]
I've done some POC in that direction in the past, integrating Widoco inside a scala-based microservice, I could share some idea, or contribute, if you want.
Anyway in my team we are using both springboot and querkus for development, and I'd suggest querkus if you are starting from scratch, as it has some very nice feature.

**Comment by seralf** [comment]
I think this is somewhat related to #450

**Comment by dgarijo** [comment]
Here, I am less interested in Docker support and more interested in having an easy to use UI. That is, double click on JAR and launches a page. I have a student looking into this, but if you have an initial idea or suggestion it is more than welcome.

**Comment by dgarijo** [comment]
of course,  previsualizing the doc at the end would be the icing on the cake
